python/ray/_private/runtime_env/pip.py

```diff
@@ -234,7 +235,9 @@ def _gen_requirements_txt():
             pip_requirements_file,
         ]
         logger.info("Installing python requirements to %s", virtualenv_path)
-        await check_output_cmd(pip_install_cmd, logger=logger, cwd=cwd, env={})
+        pip_env = os.environ.copy()
+        pip_env.update(env_vars)
+        await check_output_cmd(pip_install_cmd, logger=logger, cwd=cwd, env=pip_env)
```

Comment:
In the common case, `env_vars` is empty, so here we would be changing `env={}` to `env=os.environ.copy()`.  These two should behave exactly the same with `check_output_cmd`, right? Just want to double check.